sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -846,19 +865,37 @@ export class CertificateClient {
     const requestOptions = operationOptionsToRequestOptionsBase(options);
     const span = this.createSpan("createIssuer", requestOptions);
 
+    const generatedOptions: KeyVaultClientSetCertificateIssuerOptionalParams = requestOptions;
+
+    if (
+      options.organizationId ||
+      (options.administratorContacts && options.administratorContacts.length)
+    ) {
+      generatedOptions.organizationDetails = {
+        id: options.organizationId,
+        adminDetails: options.administratorContacts
+      };
+    }
+
+    if (options.enabled) {
```

Comment:
So if the options.enabled is false, we don't want to even set the value? We could potentially always set the value

Reply:
working on this!

Reply:
Should be good now :)